bin.js

```diff
@@ -44,6 +44,12 @@ For more details, go to https://pnpm.js.org/`))
 
 If you don't have Yarn, install it via "npm i -g yarn".
 For more details, go to https://yarnpkg.com/`))
+
```

Comment:
missing:
```js
break;
```

this bugs causes it to output both yarn and bun tips when it fails

Reply:
nice catch! I made a new PR for this issue - #25